src/V3Descope.cpp

```diff
@@ -152,8 +152,13 @@ class DescopeVisitor final : public VNVisitor {
                         }
                     }
 
-                    AstNode* const returnp = new AstCReturn(
-                        funcp->fileline(), new AstCCall(funcp->fileline(), funcp, argsp));
+                    AstCCall* const callp = new AstCCall{funcp->fileline(), funcp, argsp};
+                    if (AstNodeDType* const dtypep = funcp->dtypep()) {
+                        callp->dtypep(dtypep);
+                    } else {
+                        callp->dtypeSetVoid();
+                    }
```

Comment:
Some functions sloppily don't have dtypep (which should be fixed), but because `AstCCall` is now an `AstNodeMath`, which has `hasDType`, `V3Broken` demands it's set. I can't find `dtypepVoidIfNull`, and I'm not sure it's worth adding for this one case. Proper fix is to make sure all CFuncs have dtypep correctly set, and set the dtype of the call in the constructor from the CFunc.